subsystems/intakesubsystem.py

```diff
@@ -68,6 +69,9 @@ def periodic(self) -> None:
         frontLimitState = self.intakeMotor.getLimitSwitch(self.frontSensor)
         backLimitState = self.intakeMotor.getLimitSwitch(self.backSensor)
         self.hasPosition = backLimitState or frontLimitState
+        if not self.hasPosition:
+            self.putInPlace = False
+
         if self.state == self.IntakeState.Intaking:
```

Comment:
there's nothing in this state that tells the intake motor to run inwards, and there should also be logic for hasposition

Reply:
Intake running is done by voltage control, so it's assumed we have the inversion true/false the way that it is irl, also has position does have logic (sorta)